dense_tiles/dense_tiles.cpp

```diff
@@ -25,31 +25,42 @@ void print_help(const char* progname) {
     std::cerr << "List the (meta) tiles with the highest node density in the input file.\n";
     std::cerr << "In meta tile mode, which is the default, only coordinates for the upper\n";
     std::cerr << "left tile in an 8x8 tile block will be output.\n";
-    std::cerr << "   --help | -h           this help\n";
-    std::cerr << "   --zoom <n> | -z <n>   compute for zoom n [14]\n";
-    std::cerr << "   --max <n> | -m <n>    list n densest tiles [100000]\n";
-    std::cerr << "   --single | -s         compute for single tiles, not meta tiles\n";
-    std::cerr << "   --progress | -p       display progress bar\n";
+    std::cerr << "   --help | -h              this help\n";
+    std::cerr << "   --zoom <n> | -z <n>      compute for zoom n [14]\n";
+    std::cerr << "   --max <n> | -m <n>       list n densest tiles [100000]\n";
+    std::cerr << "   --min_nodes <n> | -M <n> list all tiles with more than n nodes;\n";
+    std::cerr << "                            only relevant if your --max setting would list\n";
+    std::cerr << "                            tiles with less than n nodes\n";
+    std::cerr << "   --all-tiles | -a         list all tiles (overwrites -m)\n";
```

Comment:
How about using `-m 0` for this instead of `-a`? This would remove the possible confusion and checks when both options are used.